pykickstart/parser.py

```diff
@@ -173,6 +176,38 @@ def next(self):
     def __next__(self):
         return self.next()                          # pylint: disable=not-callable
 
+###
+### CERTIFICATE HANDLING
+###
+class Certificate(KickstartObject):
+    """A class representing a single %certificate section."""
+    def __init__(self, *args, **kwargs):
+        """Create a new Certificate instance.  Instance attributes:
+
+           :keyword cert: The certificate to be used.
+           :keyword name: The name of the certificate.
+           :keyword path: The path where the certificate should be stored.
+        """
+        KickstartObject.__init__(self, *args, **kwargs)
+        self.cert = "\n".join(kwargs.get("cert", []))
```

Comment:
@KKoukiou wouldn't it be more simple to expect string here and join the lines before creating the object in the section?